TesseractOCR/G8Tesseract.mm

```diff
@@ -112,7 +113,10 @@ - (id)initWithLanguage:(NSString *)language
 
             _absoluteDataPath = [cachesPath stringByAppendingPathComponent:_absoluteDataPath].copy;
 
-            [self moveTessdataToCachesDirectoryIfNecessary];
+            BOOL succes = [self moveTessdataToCachesDirectoryIfNecessary];
+            if (succes == NO) {
```

Comment:
Can you fix the typo here? It should be "success". After that I'll merge, everything else looks good!